Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+        readable_output=tableToMarkdown('Object Types', outputs, headers=hr_headers, removeNull=True)
+    )
+
+
+def jira_asset_object_create_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Creates a new Jira asset object.
+    :param client: Client object for performing API requests.
+    :param args: A dictionary of command arguments:
+        - 'object_type_id': The ID of the object type to create the object for.
+        - 'attributes': A list of attribute names and values to set for the object.
+        - 'attributes_json': A JSON string of attributes to set for the object.
+    :return: A CommandResults object containing the created object as output, and a human-readable markdown table.
+    """
+    object_type_id = args.get('object_type_id')
+    attributes = args.get('attributes')
+    attributes_json = args.get('attributes_json')
+
+    json_data = get_attributes_json_data(object_type_id, attributes, attributes_json)
+    res = client.create_object(json_data)
+    outputs, object_id = parse_object_results(res).values()
+
+    return CommandResults(
+        outputs=outputs,
+        outputs_prefix=f'{INTEGRATION_OUTPUTS_BASE_PATH}.Object',
+        readable_output=f'Object created successfully with ID: {object_id}'
+    )
+
+
+def jira_asset_object_update_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Updates an existing Jira asset object.
+    :param client: Client object for performing API requests.
+    :param args: A dictionary of command arguments:
+        - 'object_id': The ID of the object to update.
+        - 'attributes': A list of attribute names and values to set for the object.
+        - 'attributes_json': A JSON string of attributes to set for the object.
+    :return: A CommandResults object containing the updated object as output, and a human-readable markdown table.
+    """
+    attributes = args.get('attributes')
+    attributes_json = args.get('attributes_json')
+    object_id = args.get('object_id')
+
+    jira_object = client.get_object(object_id)
+    object_type_id = jira_object.get('objectType').get('id')
+    json_data = get_attributes_json_data(object_type_id, attributes, attributes_json)
+    res = client.update_object(object_id, json_data)
+    _, object_id = parse_object_results(res).values()
+
+    return CommandResults(readable_output=f'Object {object_id} updated successfully')
+
+
+def jira_asset_object_delete_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Deletes an existing Jira asset object.
+    :param client: Client object for performing API requests.
+    :param args: A dictionary of command arguments:
+        - 'object_id': The ID of the object to delete.
+    :return: A CommandResults object containing a human-readable message.
+    """
+    object_id = args.get('object_id')
+
+    try:
+        client.delete_object(object_id)
+    except DemistoException as e:
+        if e.res.status_code == 404:
```

Comment:
you can pass 404 as one of the ok_codes in client and then you dont need to check this
more generally, status codes are a client implementation detail

Reply:
done